### PR TITLE
Added all the compiler blogs

### PR DESCRIPTION
**Pull request by SAtacker**
* Original Links http://sra.vjti.info/blog/compilers

### CODE REVIEW

File: _posts/2017-07-31-compiler-interpreter.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: What Is The Difference Between A Compiler And A Interpreter?
+tags: compilers interpreters
```
Comment (VedantParanjape) [inline _posts/2017-07-31-compiler-interpreter.md:4]: remove spaces from the tags use `_`, it's mentioned in the readme. If it isn't please add it.

Comment (VedantParanjape) [reply _posts/2017-07-31-compiler-interpreter.md:4]: A tag of `programming_language` makes more sense.

Comment (SAtacker) [reply _posts/2017-07-31-compiler-interpreter.md:4]: @VedantParanjape Tags are space separated [ref](https://github.com/bitbrain/blog/blame/master/_posts/2017-08-16-terminal-setup-on-macos.md#L3)

Comment (SAtacker) [reply _posts/2017-07-31-compiler-interpreter.md:4]: `programming_language` seems too long, maybe `semantics`

Comment (VedantParanjape) [reply _posts/2017-07-31-compiler-interpreter.md:4]: > @VedantParanjape Tags are space separated [ref](https://github.com/bitbrain/blog/blame/master/_posts/2017-08-16-terminal-setup-on-macos.md#L3)

TIL

**Comment by VedantParanjape** [comment]
LGTM !

**Pull request merged**